TPHost.m

```diff
@@ -217,7 +217,10 @@ - (NSImage*)defaultBackgroundImage
 	else if(osVersion < TPHostOSVersion(11)) {
 		imageName = @"10.10-Yosemite";
 	}
-	else {
+    else if(osVersion < TPHostOSVersion(12)) {
+        imageName = @"10.11-ElCapitan";
+    }
+    else {
```

Comment:
It looks like formatting is inconsistent with the rest of the file. Maybe it's worth fixing? I'm not sure how strict conventions in an Objective-C project are (I'm coming from Go where formatting issues don't happen because of `gofmt`).